packages/components/src/buttons/src/EmbeddedButton.tsx

```diff
@@ -30,6 +30,21 @@ export const GlobalEmbeddedButtonCssSelector = "hop-EmbeddedButton";
 export type EmbeddedButtonSize = "md" | "lg";
 export type EmbeddedButtonVariant = TagVariant;
 
+// TODO: Remove this when Orbiter is not used anymore
+type OrbiterVariants = "informative" | "warning";
+
+export function mapOrbiterToHopperVariants(variant: TagVariant | OrbiterVariants): TagVariant {
```

Comment:
I'm just wondering if this is the right place for it.. 
These are more Tag variants than the embedded button. but maybe it doesn't matter too much since it'll eventually be removed.